config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json

```diff
@@ -3,6 +3,15 @@
         "5.0": {
             "status": "supported",
             "latest-revision": {
+                "revision": "4",
+                "release-date": "27 January 2023",
+                "cves": "CVE-2022-43680, CVE-2022-42010, CVE-2022-42011, CVE-2022-42012, CVE-2022-3821, CVE-2022-35737, CVE-2021-46848",
+                "golang": "1.18.9",
```

Comment:
Keep `golang` field empty as in this build golang version is not getting updated